tools/post-merge

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+echo ""
+echo ""
+echo ""
+echo "#########################################################################"
```

Comment:
if you want you can source "$LISA_HOME/shell/lisa_colors" and use that to make the message red or something.

Reply:
Sure.